lib/v2/nua/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'nua.edu.cn': {
+        _name: '南京艺术学院',
+        gra: [
```

Comment:
```suggestion
        grad: [
```
The FQDN is `grad.nua.edu.cn`